### ISSUE TITLE
video autoplay: unmute action is not accessible via keyboard or screen-reader

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#12480]
because it is not a focusable button. it should be.

**Comment by ampprojectbot** [comment]
This issue seems to be in  Pending Triage for awhile. @cathyxz Please triage this to an appropriate milestone.

**Comment by aghassemi** [comment]
🎉
